pkg/controller/migcluster/remotecluster.go

```diff
@@ -50,6 +51,9 @@ func (r *RemoteClusterSource) run() {
 		}
 
 		for _, cluster := range list {
+			if r.Namespace != "" && cluster.Namespace != r.Namespace {
```

Comment:
this might not be required. The list here is filtered by openshift-migration namespace